### PR TITLE
dev upstream

### PR DESCRIPTION
**Pull request by cppvik**
* fixed `whitelisted-pods` rules

### CODE REVIEW

File: v1/k8s/namespace/allowed-images.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+evidence:
+  signed: false
+  format-type: generic
+  target_type: data
+  predicate_type: http://scribesecurity.com/evidence/discovery/v0.1
+  labels:
+    - "asset_type=namespace"
+
+with:
+  images_to_include: []
```
Comment (dn-scribe) [inline v1/k8s/namespace/allowed-images.yaml:17]: Need to document somewhere that the images are prefixes

File: v1/k8s/namespace/verify-namespace-duration.rego [hunk 2]
```diff
@@ -0,0 +1,87 @@
+package verify
```
Comment (dn-scribe) [inline v1/k8s/namespace/verify-namespace-duration.rego:1]: Not sure how useful this is.

File: v1/k8s/namespace/verify-namespace-termination.rego [hunk 3]
```diff
@@ -0,0 +1,63 @@
+package verify
```
Comment (dn-scribe) [inline v1/k8s/namespace/verify-namespace-termination.rego:1]: What is the use case of this? 
Can we specify for a specific namespace?

File: v1/k8s/namespace/white-listed-namespace.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+evidence:
+  signed: false
+  format-type: generic
+  target_type: data
+  predicate_type: http://scribesecurity.com/evidence/discovery/v0.1
+  labels:
+    - "asset_type=namespace"
+
+with:
+  white_listed_namespace: []
```
Comment (dn-scribe) [inline v1/k8s/namespace/white-listed-namespace.yaml:17]: Consider changing to `namespaces`

File: v1/k8s/namespace/white-listed-pod.rego [hunk 5]
```diff
@@ -0,0 +1,79 @@
+package verify
```
Comment (dn-scribe) [inline v1/k8s/namespace/white-listed-pod.rego:1]: We should have a way to specify "all namespaces", I did not understand this is suppported (from the code)

File: v1/k8s/pods/verify-pod-duration.yaml [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+evidence:
+  signed: false
+  format-type: generic
+  target_type: data
+  predicate_type: http://scribesecurity.com/evidence/discovery/v0.1
+  labels:
+    - "asset_type=pod"
+
+with:
+  threshold_date: "1970-01-01T00:00:00Z" # ISO 8601 string date format
```
Comment (dn-scribe) [inline v1/k8s/pods/verify-pod-duration.yaml:17]: If we have a policy for duration - we should specify number of days, and not expect the user to calculate; it is much more practical for pipline integration to use past-days and not specific dates. 
This comment is true for other date-based rules.

File: v1/k8s/pods/verify-pod-termination.yaml [hunk 7]
```diff
@@ -0,0 +1,14 @@
+name: Verify Pod Termination
```
Comment (dn-scribe) [inline v1/k8s/pods/verify-pod-termination.yaml:1]: Again: what is the use case? 
and how can I specify which pod (regex) should be terminated? 
Maybe for both namespaces and pods we should have a "verify-x-state" and give as input the desired state (terminated, running and maybe others)

File: v1/k8s/pods/white-listed-pod.yaml [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+evidence:
+  signed: false
+  format-type: generic
+  target_type: data
+  predicate_type: http://scribesecurity.com/evidence/discovery/v0.1
+  labels:
+    - "asset_type=pod"
+
+with:
+  white_listed_pod: []
```
Comment (dn-scribe) [inline v1/k8s/pods/white-listed-pod.yaml:17]: Again - how to default to all namespaces?

File: v1/k8s/namespace/verify-namespace-duration.rego (hunk 2 above)
Comment (cppvik) [reply v1/k8s/namespace/verify-namespace-duration.rego:1]: delete it?

**Review by houdini91 — APPROVED** [review]

**Pull request merged**